ReactAndroid/src/main/java/com/facebook/react/uimanager/TouchTargetHelper.java

```diff
@@ -162,25 +170,28 @@ private static boolean isTransformedTouchPointInView(
       localX = localXY[0];
       localY = localXY[1];
     }
-    if (child instanceof ReactHitSlopView && ((ReactHitSlopView) child).getHitSlopRect() != null) {
-      Rect hitSlopRect = ((ReactHitSlopView) child).getHitSlopRect();
-      if ((localX >= -hitSlopRect.left
-              && localX < (child.getRight() - child.getLeft()) + hitSlopRect.right)
-          && (localY >= -hitSlopRect.top
-              && localY < (child.getBottom() - child.getTop()) + hitSlopRect.bottom)) {
-        outLocalPoint.set(localX, localY);
-        return true;
-      }
 
-      return false;
-    } else {
-      if ((localX >= 0 && localX < (child.getRight() - child.getLeft()))
-          && (localY >= 0 && localY < (child.getBottom() - child.getTop()))) {
-        outLocalPoint.set(localX, localY);
-        return true;
-      }
+    boolean hittest = isLocalCoordinateInView(localX, localY, child);
+
+    if (allowOverflow || hittest) {
+      outTestData.set(localX, localY, hittest);
```

Comment:
```suggestion
      mTouchTestData.set(localX, localY, hittest);
```

Reply:
There was a tiny bug in the overflow test.
If you have change suggestions, please re-review them on the updated version.